### DISCUSSION TITLE
The battery of the external temperature sensor is not displayed when the connected node is secured Fibaro FGT001

### DISCUSSION THREAD

**Discussion by Marlo461** [zwave-js/zwave-js-ui#2781] [Q&A]
Please help. How can the battery status of the external temperature sensor be displayed when I connect the Thermostatic Valve to the secured S2? When I connect it without securing S2, the battery is displayed. I am attaching a print screen
![FGT001 with security S2](https://user-images.githubusercontent.com/96550831/199721383-bc1ae9f2-4194-4a05-aba5-35a6041b4321.png)
![FGT001 without security S2](https://user-images.githubusercontent.com/96550831/199721387-5de31b1f-ffff-4d91-a018-d12a5b0504be.png)

**Comment by robertsLando** [comment]
Actually only one battery can be shown, this needs to added to feat requests
